packages/myst-frontmatter/src/frontmatter/frontmatter.spec.ts

```diff
@@ -47,25 +47,28 @@ const TEST_BIBLIO: Biblio = {
 };
 
 const TEST_THEBE: Thebe = {
-  useBinder: false,
-  useJupyterLite: false,
-  requestKernel: true,
-  binderOptions: {
-    binderUrl: 'https://my.binder.org/blah',
+  lite: false,
+  binder: {
+    url: 'https://my.binder.org/blah',
     ref: 'HEAD',
     repo: 'my-repo',
-    repoProvider: 'github',
+    provider: 'github',
   },
-  serverSettings: {
-    baseUrl: 'http://localhost:8888',
-    token: 'test-secret',
-    wsUrl: 'ws://localhost:8888',
-    appendToken: true,
+  server: {
+    baseUrl: 'https://my.server.org',
+    token: 'legit-secret',
   },
-  kernelOptions: { kernelName: 'python 3', name: 'python3', path: 'some-path' },
-  savedSessionOptions: { enabled: true, maxAge: 383000, storagePrefix: 'thebe' },
+  kernelName: 'python3',
+  sessionName: 'some-path',
+  disableSessionSaving: true,
```

Comment:
I modified this from `sessionSaving` so `undefined` provides an appropriate default (the docs called for default `sessionSaving = true`, so `disableSessionSaving = undefined` is equivalent).

Reply:
yeah this is good!